sunpy/cm/color_tables.py

```diff
@@ -17,7 +18,7 @@ def _mkx(i, steps, n):
     """ Generate list according to pattern of g0 and b0. """
     x = []
     for step in steps:
-        x.extend(range(i, step + n, n))
+        x.extend(list(range(i, step + n, n)))
```

Comment:
It does, but when you are giving it to extend a list (`x`) seems to work.  I don't know whether that's normal or not...